src/main/java/neresources/nei/NEIOreHandler.java

```diff
@@ -109,6 +114,31 @@ public void drawExtras(int recipe)
     }
 
     @Override
+    public List<String> handleTooltip(GuiRecipe gui, List<String> currenttip, int recipe)
+    {
+        if (GuiContainerManager.shouldShowTooltip(gui) && currenttip.size() == 0)
+        {
+            Point offset = gui.getRecipePosition(recipe);
+            Point pos = getMousePosition();
+            Point relMouse = new Point(pos.x - gui.guiLeft - offset.x, pos.y - gui.guiTop - offset.y);
+            // Check if we are inside the coordinate system
+            if (relMouse.x > X_OFFSPRING && relMouse.x < X_OFFSPRING + X_AXIS_SIZE &&
+                relMouse.y > Y_OFFSPRING - Y_AXIS_SIZE && relMouse.y < Y_OFFSPRING)
+            {
+                CachedOre cachedOre = (CachedOre) arecipes.get(recipe);
+                float[] chances = cachedOre.oreMatchEntry.getChances();
+                double space = X_AXIS_SIZE / (chances.length * 1D);
+                // Calculate the hovered over y value
+                int yValue = (int) ((relMouse.x - X_OFFSPRING) / space);
+                if (yValue > 0 && yValue < chances.length)
+                    //TODO: The shift of one element here is due to some minor inaccuracy in the drawing function and could be avoided
```

Comment:
Please use brackets when ifs are more then one like. When it's just one no brackets are fine but with comment it looks more clean with them, gives more overview